fineract-provider/src/main/java/org/apache/fineract/portfolio/client/domain/ClientIdentifier.java

```diff
@@ -48,6 +48,9 @@
     @Column(name = "document_key", length = 1000)
     private String documentKey;
 
+    @Column(name = "document_issue_country", length = 1000)
```

Comment:
Also, let's not give the false impression that this field is going to contain an actual country name but the code so let's name it appropriately like `document_issuing_country_code`.